src/platform/imx8/include/platform/lib/dma.h

```diff
@@ -18,7 +18,11 @@
 #define DMA_ID_EDMA0	0
 #define DMA_ID_HOST	1
 
-#define dma_chan_irq(dma, chan) (dma_irq(dma) + chan)
+/* TODO Remove hardcoded interrupt values
+ * These values work for channels 6 and 7, used for ESAI
+ */
+#define dma_chan_irq(dma, chan) 58
+#define dma_chan_irq_name(dma, chan) "irqsteer6"
```

Comment:
Would you agree to comment where this value 58 is from ? Since we will have to port the SAI soon, any comment is much appreciated - even temporary patches. Thanks in advance :)

Reply:
@jlacla this number is very well explained in the commit message. But yes, @paulstelian97 please also add a comment here.